### PR TITLE
feat- Upload Modal

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**

### CODE REVIEW

File: src/pages/NewRecord.js [hunk 1]
```diff
@@ -126,9 +130,20 @@ const NewRecord = (props) => {
...
   });
 
-  function goBackToAddress() {
-    setProgress(PROGRESS_STATUSES.PATIENT_ADDRESS);
+  function uploadClickHandler() {
+    setModalState(true);
+  }
+
+  function uploadFileHandler(files) {
+    // const fileToUpload = { ...files };
```
Comment (Aniruddha-Shriwant) [inline src/pages/NewRecord.js:138]: Currently, an array of files is returned from the modal
If we need an object uncomment this and comment the next line

File: src/pages/NewRecord.js [hunk 2]
```diff
@@ -126,9 +130,20 @@ const NewRecord = (props) => {
...
+    setuploadedFiles(files);
```
Comment (Aniruddha-Shriwant) [inline src/pages/NewRecord.js:139]: Currently, an array of Files is returned from the Upload modal, but if we need an object uncomment the above line and vice-versa

Comment (Aniruddha-Shriwant) [reply src/pages/NewRecord.js:139]: + *`setuploadedFiles(fileToUpload);`

Comment (OmkarPh) [reply src/pages/NewRecord.js:139]: its good as it is ;)

**Pull request merged**